lib/controller.rb

```diff
@@ -17,39 +18,39 @@ def play_game
     @handler.display_intro_and_instructions
```

Comment:
This method is WAY too long.  Start by breaking it into smaller functions and then look for ways you can simplify.

Reply:
I'll work on this over the next few days. I want to be careful and take it step by step.